packages/main/src/OverflowToolbarButton.ts

```diff
@@ -0,0 +1,11 @@
+import { deprecationNotice } from '@ui5/webcomponents-react-base';
```

Comment:
In my opinion, you can also delete this file because it's a new component.
These files only exist for components that have been exported via this path before.